src/field_effect_helpers.c

```diff
@@ -7,6 +8,7 @@
 #include "fieldmap.h"
 #include "gpu_regs.h"
 #include "metatile_behavior.h"
+#include "oras_dowse.h"
```

Comment:
These added lines can be reverted because the code was moved to its own dedicated file